index.bs

```diff
@@ -634,6 +634,15 @@ Each {{XRSession}} has a <dfn>minimum inline field of view</dfn> and a <dfn>maxi
 
 Each {{XRSession}} has a <dfn>minimum near clip plane</dfn> and a <dfn>maximum far clip plane</dfn>, defined in meters. The values MUST be determined by the user agent and MUST be non-negative. The [=minimum near clip plane=] SHOULD be less than <code>0.1</code>. The [=maximum far clip plane=] SHOULD be greater than <code>1000.0</code> (and MAY be infinite).
 
+<div class="algorithm" data-algorithm="update-layers-state">
+
+When the user agent will <dfn>update the pending layers state</dfn> with {{XRSession}} |session| and {{XRRenderStateInit}} |newState|, it must run the following steps:
+  1. If |newState|'s {{XRRenderStateInit/layers}}'s value is not <code>null</code>, throw an {{InvalidStateError}}.
```

Comment:
@Manishearth This line will change existing behavior because if you pass a layers array today it's just ignored.
Maybe instead of throwing, we should instead say that nothing should happen (or layers removed from the array)

Reply:
Yeah, I think I mentioned this is a slight breaking change, but I don't think any content would be passing down a `layers` field. This kind of breaking change happens, to me it's similar to the breaking changes introduced when we add a new method to an interface (content could theoretically be attaching their own extension and relying on it to work)

Reply:
think this needs to be a NotSupportedError so we have full parity.

(InvalidState is for when you mix the two _with_ the layers feature enabled)

Reply:
done!